core/services/ocr2/plugins/ocr2keeper/evm21/feed_lookup.go

```diff
@@ -225,16 +229,15 @@ func (r *EvmRegistry) checkCallback(ctx context.Context, values [][]byte, lookup
 	// call checkCallback function at the block which OCR3 has agreed upon
 	err = r.client.CallContext(ctx, &b, "eth_call", args, hexutil.EncodeUint64(lookup.block))
 	if err != nil {
-		return nil, err
+		return RpcFlakyFailure, true, nil, err
```

Comment:
this can be a RPC error or the contract reverts...

Reply:
comment in code for now?

Reply:
I think it's fine to assume that the contract shouldn't revert and have all of them as retryable, but adding a todo for more thought would be good for now